powershellgroup/Tests/powershellgroup.config.tests.ps1

```diff
@@ -39,4 +39,24 @@ Describe 'PowerShellGroup resource tests' {
         $res.results.result.afterState[0].RebootRequired | Should -Not -BeNull
         $res.results.result.afterState[1].RebootRequired | Should -Not -BeNull
     }
+
+    It 'Export works on config with class-based resources' -Skip:(!$IsWindows){
```

Comment:
There shouldn't be a reason to skip on non Windows

Reply:
This export scenario still relies on PSDesiredStateConfiguration v2 (for resource discovery), which is not stable on non-windows.